### PR TITLE
[SYSSETUP] DoWriteProductOption(): Fix 'pData' copypastas. CORE-17069

### PR DESCRIPTION
**Pull request by SergeGautherie**
Addendum to 612729b09272aeb693e156de6b5991891951dbf2.

JIRA issue: [CORE-17069](https://jira.reactos.org/browse/CORE-17069)

### CODE REVIEW

**Comment by SergeGautherie** [comment]
@katahiromz, please, double-check.

**Comment by learn-more** [comment]
Please revert all unrelated stylistic changes.

File: dll/win32/syssetup/wizard.c [hunk 1]
```diff
@@ -392,21 +392,21 @@ AckPageDlgProc(HWND hwndDlg,
...
     DWORD ReportAsWorkstation;
     DWORD CSDVersion;
-} PRODUCT_OPTION_DATA, *LPPRODUCT_OPTION_DATA;
+} PRODUCT_OPTION_DATA;
 
 static const PRODUCT_OPTION_DATA s_ProductOptionData[] =
 {
     { L"Terminal Server\0", L"ServerNT", 0, 0x200 },
-    { L"\0", L"WinNT", 1, 0x300 }
+    { L"\0"               , L"WinNT"   , 1, 0x300 }
```
Comment (katahiromz) [inline dll/win32/syssetup/wizard.c:409]:
```suggestion
    { L"\0", L"WinNT", 1, 0x300 }
```

File: dll/win32/syssetup/wizard.c [hunk 2]
```diff
@@ -392,21 +392,21 @@ AckPageDlgProc(HWND hwndDlg,
 }
 
 static const WCHAR s_szProductOptions[] = L"SYSTEM\\CurrentControlSet\\Control\\ProductOptions";
-static const WCHAR s_szRosVersion[] = L"SYSTEM\\CurrentControlSet\\Control\\ReactOS\\Settings\\Version";
+static const WCHAR s_szRosVersion[]     = L"SYSTEM\\CurrentControlSet\\Control\\ReactOS\\Settings\\Version";
```
Comment (katahiromz) [inline dll/win32/syssetup/wizard.c:395]:
```suggestion
static const WCHAR s_szRosVersion[] = L"SYSTEM\\CurrentControlSet\\Control\\ReactOS\\Settings\\Version";
```

Comment (SergeGautherie) [reply dll/win32/syssetup/wizard.c]: Reverted.

File: dll/win32/syssetup/wizard.c (hunk 1 above)
Comment (SergeGautherie) [reply dll/win32/syssetup/wizard.c]: Reverted.

**Pull request merged**